src/vm.c

```diff
@@ -71,19 +71,28 @@ internal WrenForeignMethodFn VM_bind_foreign_method(
     bool isStatic,
     const char* signature) {
 
+  // For convenience, concatenate all of the method qualifiers into a single
```

Comment:
I need to add a disclaimer/copyright notice here, since this block was ripped from the Wren test codebase.